lib/doorkeeper_patches/models/application.rb

```diff
@@ -11,16 +11,22 @@ class Doorkeeper::Application
                                  allow_blank: true,
                                  message: 'Logo url must begin with https'
 
-  validates_acceptance_of :terms_of_service_accepted, if: :federal_agency,
-                                                      accept: true,
-                                                      allow_nil: false,
-                                                      message: :federal_agency_tos_required,
-                                                      tos_link: ->(values) { Rails.application.routes.url_helpers.legal_path(anchor: 'terms-of-service') }
-
-  validates_presence_of :organization, if: :federal_agency,
-                                       accept: true,
-                                       allow_nil: false,
-                                       message: :federal_agency_org_required
+  validates_acceptance_of :terms_of_service_accepted,
+    if: :federal_agency, accept: true, allow_nil: false,
+    message: :federal_agency_tos_required,
+    tos_link: ->(values) { Rails.application.routes.url_helpers.legal_path(anchor: 'terms-of-service') }
```

Comment:
No, I get that bit, I'm talking about `Rails.application.routes.url_helpers.legal_path(anchor: 'terms-of-service')`